crowbar_framework/spec/controllers/nodes_controller_spec.rb

```diff
@@ -154,6 +154,11 @@
       get :families
       assigns(:families).keys.should include(node.family.to_s)
     end
+
+    it "as json is not acceptable" do
```

Comment:
IMHO not a really useful spec. We don't really care if it responds to JSON or something different as long as a proper page gets displayed.